wagtail/wagtailadmin/edit_handlers.py

```diff
@@ -544,44 +544,66 @@ def render_as_field(self):
 class BasePageChooserPanel(BaseChooserPanel):
     object_type_name = "page"
 
-    _target_content_type = None
+    _target_content_types = None
 
     @classmethod
     def widget_overrides(cls):
         return {cls.field_name: widgets.AdminPageChooser(
-            content_type=cls.target_content_type())}
+            content_types=cls.target_content_types())}
 
     @classmethod
-    def target_content_type(cls):
-        if cls._target_content_type is None:
-            if cls.page_type:
-                try:
-                    model = resolve_model_string(cls.page_type)
-                except LookupError:
-                    raise ImproperlyConfigured("{0}.page_type must be of the form 'app_label.model_name', given {1!r}".format(
-                        cls.__name__, cls.page_type))
-                except ValueError:
-                    raise ImproperlyConfigured("{0}.page_type refers to model {1!r} that has not been installed".format(
-                        cls.__name__, cls.page_type))
-
-                cls._target_content_type = ContentType.objects.get_for_model(model)
+    def target_content_types(cls):
+        if cls._target_content_types is None:
+            if cls.page_types:
+                models = []
+
+                for page_type in cls.page_types:
+                    try:
+                        models.append(resolve_model_string(page_type))
+                    except LookupError:
+                        raise ImproperlyConfigured("{0}.page_type must be of the form 'app_label.model_name', given {1!r}".format(
```

Comment:
This error message refers to the old attribute name `page_type` instead of the new `page_types`. It should also include the index of the incorrect item. The error message a few lines below should also be fixed.

```
"{0}.page_types[{1}] must be of the form 'app_label.model_name', given {2!r}".format(
    cls.__name__, index, page_type)
```